### PR TITLE
[8.0] stock scanner : Add some locations for the configuration file

### PR DESCRIPTION
**Pull request by Garamotte**
Add the user home directory as fixed location for the sentinel.py client (in addition to the working directory).

### CODE REVIEW

**Comment by elicoidal** [comment]
:+1:

File: stock_scanner/README.rst [hunk 1]
```diff
@@ -19,6 +19,10 @@ Installation
 
 
 The "sentinel.py" specific ncurses client is available in the "hardware" directory.
+This application is a separate client, and can be run on any device.
+
+For mobile devices, like Windows Mobile or Android smart barcode scanners, we usually install it on a server, accessed though SSH.
```
Comment (pedrobaeza) [inline stock_scanner/README.rst]: s/though/through

File: stock_scanner/README.rst [hunk 2]
```diff
@@ -28,7 +32,7 @@ If you plan to use the specific "sentinel.py", you will need the "openobject-lib
    You must use openobject-library earlier than 2.0 with Odoo.
    The version 2.0 of openobject-library only implements the Net-RPC protocol, which was removed from v7.
 
-To test the module, some demo scenarii are available in the `demo` directory of the module.
+To test the module, some modules provide scenarii.
```
Comment (pedrobaeza) [inline stock_scanner/README.rst]: s/scenarii/scenario

File: stock_scanner/hardware/sentinel.py [hunk 3]
```diff
@@ -69,7 +69,15 @@ def __init__(self, stdscr):
         """
         # Read user configuration
         config = ConfigParser.SafeConfigParser(DEFAULT_CONFIG)
-        config.read(['.oerp_sentinelrc', '.openerp_sentinelrc'])
+        self.datadir = os.path.expanduser("~/")
+        config.read([
+            '.oerp_sentinelrc',
+            '.openerp_sentinelrc',
+            '.odoo_sentinelrc',
+            self.datadir + '.oerp_sentinelrc',
```
Comment (pedrobaeza) [inline stock_scanner/hardware/sentinel.py]: Use os.path.join to join paths

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

**Comment by Garamotte** [comment]
Thanks Pedro, changes done.

**Comment by alexis-via** [comment]
:+1:

**Review by lmignon — APPROVED** [review]
:+1:

**Comment by Cedric-Pigeon** [comment]
:+1:

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**